spec/features/admin/shipping_methods_spec.rb

```diff
@@ -103,9 +103,8 @@
 
       click_button I18n.t("actions.create")
 
-      expect(page).to have_no_button I18n.t("actions.create")
-      message = "Shipping method \"Teleport\" has been successfully created!"
-      expect(page).to have_flash_message message
+      expect(page).to have_content "Editing Shipping Method"
```

Comment:
At this point in the test the page loads from the "new" shipping method page to "edit", showing the new shipping method that has been created. The sticking point seems to be that page load transition, which is a common problem with feature specs. I've changed the expectation here to check for the presence of the title of the next page instead of the absence of an element from the previous page.

Reply:
Sounds good